lib/formatters/default.js

```diff
@@ -14,7 +14,7 @@ class DefaultPrinter {
     let printOptions = {
       console: this.console,
       includeTodo: options.includeTodo,
-      isInteractive: process.stdout.isTTY,
+      isInteractive: process.stdout.isTTY || process.env['IS_TTY'] === '1',
```

Comment:
What is the second check for? Forcing interactive mode in tests? Would you mind making an inline comment explaining why the fallback...

Reply:
Correct. Happy to add a comment.

Reply:
Done